common/djangoapps/util/tests/test_filters.py

```diff
@@ -0,0 +1,77 @@
+"""
+Test that various filters are fired for models/views in the student app.
+"""
+from django.test import override_settings
+from common.djangoapps.util import course
+from openedx_filters import PipelineStep
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+from openedx.core.djangolib.testing.utils import skip_unless_lms
+
+
+class TestPageURLRequestedPipelineStep(PipelineStep):
+    """
+    Utility class used when getting steps for pipeline.
+    """
+
+    def run_filter(self, url, org):  # pylint: disable=arguments-differ
+        """Pipeline step that modifies lms url requested."""
+        url = "https://lms-url-creation"
+        org = "org"
+        return {
+            "url": url,
+            "org": org,
+        }
+
+
+@skip_unless_lms
+class CourseAboutPageURLRequestedFiltersTest(ModuleStoreTestCase):
+    """
+    Tests for the Open edX Filters associated with the lms url creation process.
+    This class guarantees that the following filters are triggered during the microsite render:
+    - CourseAboutPageURLRequested
```

Comment:
This is not correct. Please, update this docstring with the correct description.